src/lib/connectors/bookshelf/LicenceVersionPurposeConditions.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const { bookshelf } = require('./bookshelf.js');
+
+module.exports = bookshelf.model('LicenceVersionPurposeConditions', {
+  tableName: 'water.licence_version_purpose_conditions',
+
+  idAttribute: 'licence_version_purpose_condition_id',
+
+  hasTimestamps: ['date_created', 'date_updated'],
+
+  licenceVersion () {
```

Comment:
I guess this `licenceVersion` relationship here is redundant as it can be discovered via the `licenceVersionPurpose` relationship.

Reply:
Can't disagree. :) I'll remove it.